### DISCUSSION TITLE
DragData change

### DISCUSSION THREAD

**Discussion by mbragg12** [cefsharp/CefSharp#4651] [General]
@amaitland Not sure if you saw this, but it broke our app.  So I put together a quick PR to fix it.  Figured someone else might run into it, so leaving a note here.  Basically if you had been getting file path data from DragData, it's no longer in the same place.
https://github.com/chromiumembedded/cef/issues/3568
The master PR
https://github.com/cefsharp/CefSharp/pull/4649
A PR for 119
https://github.com/cefsharp/CefSharp/pull/4650

It's currently failing in .net core build because of the RefAssembly.  I wasn't sure how you were handling those kinds of changes.

**Comment by amaitland** [comment]
> Not sure if you saw this, but it broke our app. So I put together a quick PR to fix it. Figured someone else might run into it, so leaving a note here. Basically if you had been getting file path data from DragData, it's no longer in the same place.
> [chromiumembedded/cef#3568](https://github.com/chromiumembedded/cef/issues/3568)

@mbragg12 Thanks 👍 Someone created a new issues a few days ago (#4643)

> It's currently failing in .net core build because of the RefAssembly. I wasn't sure how you were handling those kinds of changes.

If you delete `CefSharp.Core.Runtime.netcore.cs` then rebuild the solution locally it should recreate the file.

**Reply by mbragg12** [reply]
I update those files.  Thanks for the guidance.
